apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -421,11 +433,41 @@ private Beneficiary selectBeneWithLatestReferenceYear(List<Beneficiary> duplicat
 
     static IncludeIdentifiersMode determineIncludeIdentifiersMode(RequestDetails requestDetails) {
       String includeIdentifiersValue = requestDetails.getHeader(HEADER_NAME_INCLUDE_IDENTIFIERS);
-      if (Boolean.parseBoolean(includeIdentifiersValue) == true) {
+
+      List<String> headerValues = Arrays.asList(includeIdentifiersValue.split("\\s*,\\s*"));
```

Comment:
Is this saying "split on a comma with any amount of whitespace around it"?

Reply:
That's correct. I've added a comment to make this more readable.